package.json

```diff
@@ -81,6 +81,7 @@
     "style-loader": "^2.0.0",
     "stylelint": "^13.13.1",
     "stylelint-config-bitcrowd": "^4.0.0",
+    "stylelint-scss": "^3.20.1",
```

Comment:
Maybe mention in the README how to best run this (I guess it also adds an `npm` task)

Reply:
There’s no new npm script needed — this is a plugin for stylelint that adds some sass-related/useful rules to the default set provided by stylelint. So when you run `yarn lint`, these rules are used too

Reply:
…there was no section about linting there though, so I added one ✨